apps/storybook/src/NotificationMarker.test.ts

```diff
@@ -0,0 +1,16 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+describe('NotificationMarker', () => {
+  const storyPath = 'Core/NotificationMarker';
+  const tests = ['Basic', 'Pulsing', 'Status'];
+
+  tests.forEach((testName) => {
+    it(testName, function () {
+      const id = Cypress.storyId(storyPath, testName);
+      cy.visit('iframe', { qs: { id } });
+      cy.compareSnapshot(testName);
```

Comment:
I could not fix the NotificationMarker test images. In the `NotificationMarker -> Status` story for example, the notification marker circle appears on the top right when viewing in Storybook.

![image](https://user-images.githubusercontent.com/45748283/207117469-16148deb-918f-4965-8cf9-a2b0ae7f9035.png)

But the test images show the circle on the top center for the `IconButton`.

![image](https://user-images.githubusercontent.com/45748283/207117359-efa70977-a071-44e9-b500-bb67546531ca.png)

<!--![image](https://user-images.githubusercontent.com/45748283/207117291-f5393aac-349e-4334-9060-8003f790cd8e.png)-->

I tried using @mayank99 's suggestion to use `cy.wait()` but that didn't work. Does anyone have any ideas on why this is happening and how to fix this?

Reply:
After spending some time on this, I was able to narrow down the issue to `IconButton` but couldn't really figure out the exact reason it gets misaligned like that. Could be a bug in Chrome 94 (which is what Cypress 9.6.0 comes with).

I was able to work around it by removing `IconButton` and instead adding the fill/size attributes on the svg (which will not be necessary after https://github.com/iTwin/iTwinUI/issues/851 and/or https://github.com/iTwin/iTwinUI-icons/issues/55). I've pasted all my local changes below, feel free to copy/change as you see fit. Hopefully this unblocks you for now.

<details>

<summary>NotificationMarker.stories.tsx</summary>

```tsx
/*---------------------------------------------------------------------------------------------
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
import { Story, Meta } from '@storybook/react';
import React from 'react';
import {
  Text,
  NotificationMarker,
  NotificationMarkerProps,
} from '@itwin/itwinui-react';
import { SvgNotification } from '@itwin/itwinui-icons-react';

export default {
  component: NotificationMarker,
  args: {
    enabled: true,
    pulsing: false,
  },
  argTypes: {
    className: { control: { disable: true } },
    style: { control: { disable: true } },
    status: {
      control: 'radio',
      options: ['primary', 'positive', 'warning', 'negative'],
    },
  },
  title: 'Core/NotificationMarker',
  decorators: [
    (Story) => (
      <div
        style={{
          display: 'grid',
          placeItems: 'start',
          gap: '20px',
        }}
      >
        <Story />
      </div>
    ),
  ],
} as Meta<NotificationMarkerProps>;

export const Basic: Story<NotificationMarkerProps> = (args) => {
  return (
    <>
      <NotificationMarker {...args}>
        <SvgNotification width={16} height={16} fill='currentColor' />
      </NotificationMarker>
      <NotificationMarker {...args}>Normal</NotificationMarker>
    </>
  );
};

Basic.args = {
  status: 'primary',
  children: <SvgNotification />,
};

Basic.argTypes = {
  children: {
    control: false,
  },
};

export const Pulsing: Story<NotificationMarkerProps> = (args) => {
  return (
    <>
      <NotificationMarker status='primary' {...args}>
        <SvgNotification width={16} height={16} fill='currentColor' />
      </NotificationMarker>

      <NotificationMarker status='primary' {...args}>
        <Text style={{ fontWeight: '700' }}>URGENT</Text>
      </NotificationMarker>
    </>
  );
};

Pulsing.args = {
  status: 'negative',
  pulsing: true,
};

export const Status: Story<NotificationMarkerProps> = (args) => {
  return (
    <>
      <NotificationMarker status='primary' {...args}>
        <SvgNotification width={16} height={16} fill='currentColor' />
      </NotificationMarker>
      <NotificationMarker status='positive' {...args}>
        <SvgNotification width={16} height={16} fill='currentColor' />
      </NotificationMarker>
      <NotificationMarker status='warning' {...args}>
        <SvgNotification width={16} height={16} fill='currentColor' />
      </NotificationMarker>
      <NotificationMarker status='negative' {...args}>
        <SvgNotification width={16} height={16} fill='currentColor' />
      </NotificationMarker>
      <NotificationMarker status='primary' {...args}>
        Primary
      </NotificationMarker>
      <NotificationMarker status='positive' {...args}>
        Positive
      </NotificationMarker>
      <NotificationMarker status='warning' {...args}>
        Warning
      </NotificationMarker>
      <NotificationMarker status='negative' {...args}>
        Negative
      </NotificationMarker>
    </>
  );
};

Status.argTypes = {
  status: {
    control: false,
  },
};
```
</details>

Reply:
Thanks. I ended up adding a story called `NotificationMarker -> Basic` that directly uses an SVG (e.g. `<SvgNotification ... />`).

The other three stories in `NotificationMarker` use `IconButton`/`Button` and so the visual tests still give the above-mentioned bug. Thus, these three stories are left out of the visual tests.

The visual test for `Button -> WithNotification` is excluded from visual tests for the same reason.

Reply:
**[DONE]** Added comments in `NotificationMarker.test.ts` and `Button.test.ts` explaining why certain tests are left out